src/services/api.ts

```diff
@@ -661,6 +701,26 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
             },
         );
     }
+
+    restartPDiskOld({nodeId, pDiskId}: {nodeId: number | string; pDiskId: number | string}) {
+        const pDiskPath = createPDiskDeveloperUILink({
+            nodeId,
+            pDiskId,
+            host: this.getPath(''),
+        });
+
+        return this.post<ModifyDiskResponse>(
```

Comment:
This response type isn't `ModifyDiskResponse`

Reply:
it's compatible https://github.com/ydb-platform/ydb-embedded-ui/pull/821/files#diff-36e179fa73b274d07721772a28ff41e32262ca486da9ab109b912f8246d85c2eR1